dietpi/patch_file

```diff
@@ -2941,6 +2941,34 @@ _EOF_
 
 		fi
 		#-------------------------------------------------------------------------------
+		if [ grep 'aSOFTWARE_INSTALL_STATE[86]=2' /Dietpi/dietpi/.installed ] && (( $DISTRO > 3 )); then
```

Comment:
You'll need to escape the ```[]```, and ```/DietPi```
```
root@DietPi:~# grep 'aSOFTWARE_INSTALL_STATE\[74\]=0' /DietPi/dietpi/.installed
aSOFTWARE_INSTALL_STATE[74]=0

```

Reply:
Fixed and tested. Now the syntax works.

How to set if grep into brackets? [ ] and (( )) are not allowed here. Left it without now, which works, but especially with more than one condition, brackets would enhance the overview.